nni/retiarii/operation_def/torch_op_def.py

```diff
@@ -494,14 +494,25 @@ class ToDevice(PyTorchOperation):
     def __init__(self, type_name: str, parameters: Dict[str, Any], _internal: bool = False):
         self.type = "ToDevice"
         self.device = parameters['device']
+        self.overridden_device_repr = None
         self.src = parameters['src']
         self.dst = parameters['dst']
 
+    def override_device_repr(self, device_repr):
+        self.overridden_device_repr = device_repr
+
     def __repr__(self):
-        return f'to("{self.device}")'
+        if self.overridden_device_repr == None:
+            return f'to("{self.device.device_repr()}")'
```

Comment:
in what cases that `overridden_device_repr` is None?

Reply:
CUDA `GPUDevice` may remap GPU physical ID to CUDA ID. The device_repr is different from `GPUDevice.device_repr`. 
`override_device_repr` will be called in `pytorch.graph_to_pytorch_model` to replace `device_repr` with the correct
CUDA ID, e.g., when a job uses Physical GPU-1,2, its CUDA ID should be `cuda:0` and `cuda:1`.
`self.device.device_repr()` would return `cuda:1` and `cuda:2`, but `override_device_repr` should be `cuda:0` and
`cuda:1`. 

I just add the comments in code to explain this.